tools/configure_server_zone.rb

```diff
@@ -0,0 +1,29 @@
+if ARGV.empty?
```

Comment:
Maybe this should be named `change_server_zone.rb`.  We're not really configuring the zone itself.